bot/exts/moderation/infraction/superstarify.py

```diff
@@ -134,6 +135,9 @@ async def superstarify(
         if await _utils.get_active_infraction(ctx, member, "superstar"):
             return
 
+        # Set the duration to 1 hour if none was provided
+        duration = duration or datetime.datetime.utcnow() + datetime.timedelta(hours=1)
```

Comment:
```suggestion
        duration = duration or await Duration().convert(ctx, "1h")
```
You can have the `1h` as a constant at the top of the file.

Reply:
make sure to import `Duration`